index.js

```diff
@@ -210,6 +215,8 @@ function fastifyView (fastify, opts, next) {
 
   function readCallback (that, page, data) {
     return function _readCallback (err, html) {
+      const requestedPath = that?.request?.context?.config?.url
```

Comment:
This syntax is not supported by the actual node10 CI

Reply:
Note that `request.context.config` should always exist
https://github.com/fastify/fastify/blob/70749bcc278ae9c4d1487ffc1bdecd67fb8bc14e/lib/fourOhFour.js#L127
https://github.com/fastify/fastify/blob/70749bcc278ae9c4d1487ffc1bdecd67fb8bc14e/lib/route.js#L226-L230

Reply:
Thanks for feedback. I removed optional chaining. Removing check on request existence causes errors on tests execution, in order to avoid these I added manual check.